lib/jekyll-feed.rb

```diff
@@ -12,6 +12,14 @@ def config
       @context.registers[:site].config
     end
 
+    def path
+      if config["feed"] && config["feed"]["path"]
```

Comment:
The only thing that matters here is if this is supposed to be a hash, if it is then let it blow up because that's your fault and it's the right thing to do, if accepts any value then this matters.